src/inmanta/deploy/state.py

```diff
@@ -80,7 +80,7 @@ def is_dirty(self) -> bool:
 
 
 @dataclass(frozen=True)
-class ResourceDetails:
+class ResourceIntent:
```

Comment:
rename 4.

motivation: conflict with `executor.ResourceDetails` + `ResourceDetails` simply isnt't very revealing. `ResourceIntent` more clearly indicates what the object represents.

Reply:
I like this a lot

Reply:
I like it.